xml/System.Net/FileWebResponse.xml

```diff
@@ -290,7 +290,7 @@
         <Parameter Name="disposing" Type="System.Boolean" Index="0" FrameworkAlternate="netframework-1.1" />
       </Parameters>
       <Docs>
-        <param name="disposing">To be added.</param>
+        <param name="disposing"><see langword="true" /> to dispose; otherwise, <see langword="false" />.</param>
```

Comment:
```suggestion
        <param name="disposing">
          <see langword="true" /> to release both managed and unmanaged resources; otherwise, <see langword="false" />.</param>
```